backend/src/test/java/com/gildedrose/ItemsControllerTest.java

```diff
@@ -55,4 +55,16 @@ void returnsNotFoundResponse_WhenSingleItemIsNotFound() {
 
         assertEquals(Response.Status.NotFound, response.getStatus());
     }
+
+    @Test
+    void okAllItemsUpdateHandler_ShouldAgeAllItemsInDB() {
+        Request request = new Request("/inventory/update", Request.Method.POST.asString, "foo", "foo", "foo");
+        Ageable item = mockedDB.get(0);
+
+        itemsController.okAllItemsUpdateHandler.apply(request);
+
+
+        assertEquals(13, item.getQuality());
```

Comment:
How do I know the items are updated?

Reply:
We added some local variables to help explain the test in this commit [254e721](https://github.com/jramnani/GildedRoseJavaLLC/pull/34/commits/254e7217ff5a53d3b66710178d4cb34517563f41)

```
    void okAllItemsUpdateHandler_ShouldAgeAllItemsInDB() {
        Request request = new Request("/inventory/update", Request.Method.POST.asString, "foo", "foo", "foo");
        Ageable item = mockedDB.get(0);
        int defaultItemAgeDecrementor = 1;

        int expectedQuality = item.getQuality() - defaultItemAgeDecrementor;
        int expectedSellin = item.getSellIn() - defaultItemAgeDecrementor;
        itemsController.okAllItemsUpdateHandler.apply(request);
        
        assertEquals(expectedQuality, item.getQuality());
        assertEquals(expectedSellin, item.getSellIn());
    }
```

Reply:
Thanks!